cypress-tags.js

```diff
@@ -71,31 +71,29 @@ const paths = glob
   })
   .filter((pathName) => pathName.endsWith(".feature"));
 
-const featuresToRun = [];
-
-paths.forEach((featurePath) => {
+const featuresToRun = paths.filter((featurePath) => {
   const spec = `${fs.readFileSync(featurePath)}`;
-  const parsedFeature = new Parser().parse(spec);
+  const { feature } = new Parser().parse(spec);
 
-  if (!parsedFeature.feature) {
+  if (!feature) {
     debug(`Feature: ${featurePath} is empty`);
-    return;
+    return false;
   }
 
-  const featureTags = parsedFeature.feature.tags;
-  const featureShouldRun = shouldProceedCurrentStep(featureTags, envTags);
-  const taggedScenarioShouldRun = parsedFeature.feature.children.some(
-    (section) =>
-      section.tags &&
-      section.tags.length &&
-      shouldProceedCurrentStep(section.tags.concat(featureTags), envTags)
-  );
-  debug(
-    `Feature: ${featurePath}, featureShouldRun: ${featureShouldRun}, taggedScenarioShouldRun: ${taggedScenarioShouldRun}`
+  const shouldRun = feature.children.some((scenario) =>
+    scenario.examples
+      ? scenario.examples.some((example) =>
+          shouldProceedCurrentStep(
+            [...example.tags, ...scenario.tags, ...feature.tags],
```

Comment:
I discovered today in another project that null checking `section.tags` is necessary. I'll update the PR in a few days.